nav2_map_server/include/nav2_map_server/map_2d/map_server_2d.hpp

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2020 Shivam Pandey pandeyshivam2017robotics@gmail.com
+// Copyright (c) 2020 Samsung Research Russia
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//
+// Created by shivam on 9/10/20.
+//
+
+#ifndef NAV2_MAP_SERVER__MAP_2D__MAP_SERVER_2D_HPP_
+#define NAV2_MAP_SERVER__MAP_2D__MAP_SERVER_2D_HPP_
+
+#include <string>
+#include <memory>
+
+#include "nav2_map_server/map_server_core.hpp"
+#include "nav2_map_server/map_2d/map_io_2d.hpp"
```

Comment:
In the header, nothing but this will be used in the source.

Reply:
It is already included in the source file. This is duplication.
Moreover, header-file should not include unnecessary information. Imagine, someone will later completely re-work `map_io_2d.cpp` file and refuse from MapIO library usage (for some reasons). For this imaginary developer is easy to forget to remove this include, and it will be orphaned, increasing only the load of GCC preprocessor. So, if we can not rely that MapIO will always be the state of art lib in the future. So, please remove it from here.

Reply:
Great point, got it!